_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
+- K8s Objects
+- Metadata
+- Configuration
+- Security
+- Networking
+- Storage
+- Cluster Operation
+- Workloads
+- Architecture
+- API
+- Logging
```

Comment:
that makes sense---if we need to be more fine-grained later we can add those tags back in; for now I'm okay with having "Operations" cover it all